client/src/app/contact/page.tsx

```diff
@@ -0,0 +1,41 @@
+"use client"
+import Ph4 from '@/assets/ph4-contact.svg';
+import Image from 'next/image';
+
+const contact = () => {
```

Comment:
The Page and component name should starts with capital letter else it will fails the build.